internal/handlers/telegram/telegram.go

```diff
@@ -14,7 +14,7 @@ type Client struct {
 	api       *tgbotapi.BotAPI
 	Settings  internal.TelegramSettings
 	logger    internal.DebugLogger
-	sendToIrc func(string)
+	SendToIrc func(string)
```

Comment:
I had to capitalize the SendToIrc inorder to assign it outside the telegram namespace for testing. 

I dont think this is the proper way, but I dont know any other way